{"kind":"symbol","schemaVersion":{"patch":0,"minor":3,"major":0},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/validatablekit\/validationerror\/description"]}],"hierarchy":{"paths":[["doc:\/\/ValidatableKit\/documentation\/ValidatableKit","doc:\/\/ValidatableKit\/documentation\/ValidatableKit\/ValidationError"]]},"sections":[],"metadata":{"title":"description","roleHeading":"Instance Property","externalID":"s:14ValidatableKit15ValidationErrorV11descriptionSSvp","symbolKind":"property","role":"symbol","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"description","kind":"identifier"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"}],"modules":[{"name":"ValidatableKit"}]},"abstract":[{"type":"text","text":"A textual representation of this error."}],"primaryContentSections":[{"declarations":[{"platforms":["Linux"],"languages":["swift"],"tokens":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"description"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":" { "},{"kind":"keyword","text":"get"},{"kind":"text","text":" }"}]}],"kind":"declarations"}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/ValidatableKit\/documentation\/ValidatableKit\/ValidationError\/description"},"references":{"doc://ValidatableKit/documentation/ValidatableKit":{"url":"\/documentation\/validatablekit","identifier":"doc:\/\/ValidatableKit\/documentation\/ValidatableKit","title":"ValidatableKit","kind":"symbol","type":"topic","abstract":[{"type":"text","text":"Composable data validation API in Swift exposing simple DSL for writing validations."}],"role":"collection"},"doc://ValidatableKit/documentation/ValidatableKit/ValidationError":{"navigatorTitle":[{"kind":"identifier","text":"ValidationError"}],"url":"\/documentation\/validatablekit\/validationerror","fragments":[{"text":"struct","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"ValidationError"}],"type":"topic","identifier":"doc:\/\/ValidatableKit\/documentation\/ValidatableKit\/ValidationError","title":"ValidationError","kind":"symbol","role":"symbol","abstract":[{"text":"An error type representing validation failure.","type":"text"}]},"doc://ValidatableKit/documentation/ValidatableKit/ValidationError/description":{"kind":"symbol","role":"symbol","title":"description","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"description"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"}],"abstract":[{"type":"text","text":"A textual representation of this error."}],"type":"topic","identifier":"doc:\/\/ValidatableKit\/documentation\/ValidatableKit\/ValidationError\/description","url":"\/documentation\/validatablekit\/validationerror\/description"}}}